### DISCUSSION TITLE
"dd" isnt working since i last updated

### DISCUSSION THREAD

**Discussion by directormac** [LazyVim/LazyVim#936] [Q&A] [answered]
![image](https://github.com/LazyVim/LazyVim/assets/5866196/8f44d136-b1e0-4541-8585-98f3ac516621)

``  Error  8:45:41 pm msg_show.emsg E5108: Error executing lua: Vim:E15: Invalid expression: <Cmd>lua require("which-key").show("d", {mode = "n", auto = true})<CR>
stack traceback:
[C]: at 0x7ff718f623d0`

is it just me?

**Comment by folke** [comment, marked as answer]
I'm afraid it's just you. No idea why you get that error. That expression is also 100% valid and has been used since I created which-key. 

I didn't make any recent changes to which-key either.

**Comment by directormac** [comment]
Well i found out its because of hardtime.nvim updated 4hours ago. thank you folke for the response.

**Reply by folke** [reply]
yeah, hardtime causes issues with multiple keybindings in lazyvim I'm afraid. Other things may break.

This is my personal hardtime ;) Just execute that function in your `lua/config/keymaps.lua`

```lua
function M.cowboy()
  ---@type table?
  local id
  for _, key in ipairs({ "h", "j", "k", "l", "+", "-" }) do
    local count = 0
    local timer = assert(vim.loop.new_timer())
    vim.keymap.set("n", key, function()
      if vim.v.count > 0 then
        count = 0
      end
      if count >= 10 then
        id = vim.notify("Hold it Cowboy!", vim.log.levels.WARN, {
          icon = "🤠",
          replace = id,
          keep = function()
            return count >= 10
          end,
        })
      else
        count = count + 1
        timer:start(2000, 0, function()
          count = 0
        end)
        return key
      end
    end, { expr = true, silent = true })
  end
end

**Reply by directormac** [reply]
Ahhhhhhh You never disappoint! thank you for this!!!
